Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/library/centos:8.2.2004
+FROM quay.io/fedora/fedora:34
```

Comment:
Thoughts on using fedora/fedora:latest? This way we need not change in case we need a new image.

Reply:
Pinning a dependency's version with either a version tag or an image SHA will help prevent a breaking change from sneaking into the container image.

Reply:
Makes sense.